test/release/examples/primers/modules.chpl

```diff
@@ -243,6 +243,18 @@ module MainModule {
       // writeln(modToUse.bar); // would be an error, modToUse not visible
     }
 
+    /* This syntax can also be used by ``use`` statements to enable just
```

Comment:
For some reason I am not seeing this rendered in your link.

Would you be able to fix what appears as

> “`use` statement” 
> “`import` statement”

in the "Access From Outside a Module" section? These should probably use italics or something rather than quotes. E.g. *use statement*.

Reply:
"This syntax can also be used by ``use`` statements" seems unnecessarily to use both meanings of "use".

Maybe "With this syntax, a ``use`` statement can enable just ..."